platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineManager.java

```diff
@@ -0,0 +1,172 @@
+package com.mapbox.mapboxsdk.offline;
+
+import android.content.Context;
+import android.os.Handler;
+import android.os.Looper;
+import android.support.annotation.NonNull;
+
+import java.io.File;
+
+/**
+ * The offline manager is the main entry point for offline-related functionality.
+ * It'll help you list and create offline regions.
+ */
+public class OfflineManager {
+
+    // Default database name and size (1GB)
+    private final static String OFFLINE_DATABASE_NAME = "mbgl-offline.db";
+    private final static long OFFLINE_MAX_CACHE_SIZE = 1024 * 1024 * 1024;
```

Comment:
Gotcha. Clarified and corrected with https://github.com/mapbox/mapbox-gl-native/commit/3e4f8f758ec2e424c7b1a9cf9897703298183aec, I'm using the default value in `src/mbgl/util/constants.cpp`.